src/main/resources/assets/js/pages/produtos.js

```diff
@@ -10,13 +10,13 @@ export default {
         
         if(seletorDeProdutos.get().length < 1) {
           return;
-        }
+        };
     
-        const opcaoSelecionada = seletorDeProdutos.find('option:selected');
-        const unidadeMedidaElement = $('#unidadeMedida');
-        const unidadeMedida = opcaoSelecionada.attr('data-unidade-medida'); 
-    
-        unidadeMedidaElement.text(unidadeMedida);
+        const textoUnidadeMedida = seletorDeProdutos.find('option:selected');
```

Comment:
Esse `;` no final da linha tá estranho pq separa o target do método abaixo. Deviamos remover.

Reply:
força do hábito;
Retiramos